docs/tutorials/cli-tool.md

```diff
@@ -46,17 +46,17 @@ You may need to prefix these commands (`sudo npm link`) to run them.
 
 ## Launching the CLI tool
 
-1. Navigate to the `/core` folder: `cd core`
+1. Navigate to the `packages/core` folder: `cd packages/core`
 2. If on Kovan testnet, apply network addresses:
 
 ```bash
-npx apply-registry
+yarn load-addresse
 ```
```

Comment:
nit: if we're assuming they've already compiled the contracts using `qbuild`, the contracts should already be pre-loaded with the kovan addresses -- so I _think_ this means we can entirely remove this step.